tests/js/client/server_parameters/legacy-sorting-cluster.js

```diff
@@ -0,0 +1,200 @@
+                 name:"hash_unique"});
+  c.ensureIndex({type:"skiplist", fields: [attrA, attrB], unique: false,
+                 name:"skiplist_non_unique"});
+  c.ensureIndex({type:"skiplist", fields: [attrA, attrB], unique: true,
+                 name:"skiplist_unique"});
+  let nr = 6;
+  try {
+    c.ensureIndex({type:"mdi-prefixed", fields: ["x", "y"], prefixFields:[attrA, attrB],
+                   name:"mdi_non_unique", fieldValueTypes: "double", unique: false});
+    c.ensureIndex({type:"mdi-prefixed", fields: ["x", "y"], prefixFields:[attrA, attrB],
+                   name:"mdi_unique", fieldValueTypes: "double", unique: true});
+    nr += 2;
+  } catch(e) {
+    // Smile it away on 3.11
+  }
+  db._useDatabase("_system");
+  return nr;
+}
+
+function legacySortingTestSuite() {
+  'use strict';
+  const dn = "UnitTestsDatabase";
+  const cn = 'UnitTestsLegacySortingBad';
+  const cn2 = 'UnitTestsLegacySortingGood';
+
+  return {
+    setUp: function() {
+      let l = [];
+      for (let i = 0; i < 1000; ++i) {
+        l.push({Hallo:i, a:i, b: i, x: i * 1.234567, y : i * 1.234567});
+      }
+      // Note: We must only use one shard, since otherwise the incriminating
+      // documents might be distributed to more than one dbserver, removing
+      // the problem!
+      let c = db._create(cn, {numberOfShards:1, replicationFactor:2});
+      c.insert(l);
+      c = db._create(cn2, {numberOfShards:1, replicationFactor:2});
+      c.insert(l);
+
+      db._createDatabase(dn);
+      db._useDatabase(dn);
+      c = db._create(cn, {numberOfShards:1, replicationFactor:2});
+      c.insert(l);
+      c = db._create(cn2, {numberOfShards:1, replicationFactor:2});
+      c.insert(l);
+      db._useDatabase("_system");
+    },
+
+    tearDown: function () {
+      db._drop(cn);
+      db._drop(cn2);
+      db._dropDatabase(dn);
+    },
+
+    testMigrationCheck: function() {
+      let nr = createVPackIndexes("_system", cn, "a", "b");
+      createVPackIndexes(dn, cn, "a", "b");
+      createVPackIndexes("_system", cn2, "a", "b");
+      createVPackIndexes(dn, cn2, "a", "b");
+      poisonCollection("_system", cn, "a", "b");
+      poisonCollection(dn, cn, "a", "b");
+
+      let c = db._collection(cn);
+      let indexes = c.getIndexes();
+      let names = indexes.map(x => x.name);
+      let ids_system = indexes.map(x => x.id);
+      let r = arango.GET("/_admin/cluster/vpackSortMigration/check");
+
+      db._useDatabase(dn);
+      c = db._collection(cn);
+      indexes = c.getIndexes();
+      let ids_dn = indexes.map(x => x.id);
+      db._useDatabase("_system");
+
+      r = arango.GET("/_admin/cluster/vpackSortMigration/check");
+
+      // Now check that we found all the indexes and not too many.
+      // We do not check the details of the output, since we do this
+      // in the single server case and we never know which shards land
+      // on which dbserver. We do not even know
+      assertEqual(false, r.error);
```

Comment:
I suggest to replace all `assertEqual(false, ...)` with `assertFalse(...)` and all `assertEqual(true, ...)` with `assertTrue(...)`.

Reply:
Done.